plugins/content_distribution/providers/tvinci/TvinciDistributionPlugin.php

```diff
@@ -191,8 +198,15 @@ public static function getKalturaProvider()
 	 */
 	public static function contributeMRSS(EntryDistribution $entryDistribution, SimpleXMLElement $mrss)
 	{
+
+	}
+
+	public function getObjectFeatureType ()
```

Comment:
Lets discuss both functions  getObjectFeatureType  & contribute if we need them at all, since the parent function will be called if they are gone.